packages/space-opera/src/components/ibl_selector/ibl_selector.ts

```diff
@@ -164,15 +111,10 @@ export class IblSelector extends ConnectedLitElement {
       return;
     }
 
-    const filename = (files[0] as File).name;
-
-    const arrayBuffer = await files[0].arrayBuffer();
-    const safeObjectUrl = createSafeObjectUrlFromArrayBuffer(arrayBuffer);
-    const unsafeUrl = filename.match(/\.(hdr)$/) ?
-        safeObjectUrl.unsafeUrl + '#.hdr' :
```

Comment:
My hack is gone! 😢😁

Reply:
What hack?

Reply:
Oh it's still there - I factored it into createBlobUrlFromEnvironmentImage - that what you mean?

Reply:
https://github.com/google/model-viewer/blob/07bd0edceff1f61f22fa3b0c1db3aae809392fb8/packages/space-opera/src/redux/lighting_dispatchers.ts#L43

Yesss! Definitely my best contribution to the project 😁

Reply:
ah, now I see; nice...